### PR TITLE
Implement "activity" sorting in Account search

### PR DESCRIPTION
**Pull request by znarf**
Related to https://github.com/opencollective/opencollective/issues/5375

Related Front End PR: https://github.com/opencollective/opencollective-frontend/pull/7637

### CODE REVIEW

**Review by SudharakaP — COMMENTED** [review]
@znarf : Should we add the sorting by creation date to this PR as well since we have the [same issue](https://github.com/opencollective/opencollective/issues/5375) for that?

File: server/graphql/v2/query/collection/AccountsQuery.ts [hunk 1]
```diff
@@ -59,6 +58,7 @@ const AccountsQuery = {
       const cleanTerm = args.searchTerm.trim();
 
       const extraParameters = {
+        orderBy: args.orderBy || { field: 'rank', direction: 'DESC' },
```
[the remark this answers is not in the document]
Comment (znarf) [reply server/graphql/v2/query/collection/AccountsQuery.ts:61]: If activity is a number, does it make sense to sort by "most recent activity" or "oldest activity"?

[the remark this answers is not in the document]
Comment (SudharakaP) [reply server/graphql/v2/query/collection/AccountsQuery.ts:61]: That makes sense actually. I was looking at the design and somehow have this concept in my mind that activity should be ascending and descending. But then given that we based it on `netAmountInCollectiveCurrency` I do see that there cannot be a concept of "most recent" and "oldest activity"; so we will just have activity. 🤔

File: server/lib/search.js [hunk 2]
```diff
@@ -167,18 +168,30 @@ export const searchCollectivesInDB = async (
   }
 
   // Build the query
+
+  const sortSubqueries = {
+    ACTIVITY: `
+      SELECT COALESCE(SUM(ABS("netAmountInCollectiveCurrency")), 0)
```
Comment (Betree) [inline server/lib/search.js:174]: `COUNT` could be a better metric than `SUM` IMO. With 'SUM`, anyone adding a fake $1000000 through added funds will arrive at the top of the page directly. It's easier to fake amounts than the number of transactions.

**Pull request merged**